.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "parser": "babel-eslint"
```

Comment:
I already added eslint config [as `eslintConfig` within package.json](https://github.com/edx/studio-frontend/blob/master/package.json#L49-L61), which is what create react app does. I was thinking that would be an easier central location than an .eslintrc, but now that our config is growing in size, I'm thinking it might be better to just dump it all into this eslintrc. What do you think @thallada ?

Reply:
Oh I didn't even see that. I can move that config here. Also, I think I'm going to rename this to `.eslintrc.json` selfishly just because it makes my editor know to syntax highlight it as json.